qpup_hw/config/qpup_hardware.yaml

```diff
@@ -0,0 +1,47 @@
+qpup:
+  combined_robot_hw:
+    shared_values:
+      splay_transmission_reduction: &shared_splay_transmission_reduction 40
+      hip_transmission_reduction: &shared_hip_transmission_reduction 40
+      knee_transmission_reduction: &shared_knee_transmission_reduction deg(rad(40 * 63 / 39))
```

Comment:
Why `deg(rad(...))`, just to force an evaluation? Also does this int division truncate? 96.9230769 -> 96 is not good

Reply:
just to force the evaluation. it def evaluates as float tho. the contents is evaluated in python